src/queryBuilder/createOne.ts

```diff
@@ -10,6 +10,16 @@ export class CreateOneBuilder<TCreateFields extends Record<string, any>> extends
     this._data = data
   }
 
+  /**
+   * Publishes and event notifying subscriber of Change
+   */
+  public publish(): this {
+    if (this._options.pubSub) {
```

Comment:
suggestion: Add tests to cover both branches of this `if` statement

You can view test coverage issues by clicking on the "Details" link under the failing Coveralls check

<img width="886" alt="Screen Shot 2020-07-04 at 8 33 50 PM" src="https://user-images.githubusercontent.com/18018864/86523215-1ac7c580-be37-11ea-91a6-a31f512637b1.png">

Click on the "CHANGED" tab to view changed files

<img width="508" alt="Screen Shot 2020-07-04 at 8 34 11 PM" src="https://user-images.githubusercontent.com/18018864/86523217-1f8c7980-be37-11ea-82b8-e209eaa69df5.png">

Files with less coverage are shown in red

<img width="252" alt="Screen Shot 2020-07-04 at 8 34 21 PM" src="https://user-images.githubusercontent.com/18018864/86523219-23200080-be37-11ea-815e-d59f6e8a1914.png">

Click the file link for a detailed view with problem areas highlighted

<img width="1102" alt="Screen Shot 2020-07-04 at 8 34 41 PM" src="https://user-images.githubusercontent.com/18018864/86523222-261af100-be37-11ea-9d92-0dd479b8f64c.png">

The current test only exercises this method when `this._options.pubSub` is truthy. We need to also account for scenarios where that's not the case. This is also something that's better handled through unit tests than the integration tests.